Entities/Common/Controls/StandardControls.as

```diff
@@ -357,39 +363,17 @@ void ManageCamera(CBlob@ this)
 	CControls@ controls = this.getControls();
 
 	// mouse look & zoom
-	if ((getGameTime() - this.get_s32("tap_time") > 5) && controls !is null)
+	if ((getGameTime() - this.get_s32("tap_time") > 2) && controls !is null)
 	{
 		if (controls.isKeyJustPressed(controls.getActionKeyKey(AK_ZOOMOUT)))
 		{
-			if (zoomLevel == 2)
-			{
-				zoomLevel = 1;
-			}
-			else if (zoomLevel == 1)
-			{
-				zoomLevel = 0;
-			}
-			else if (zoomLevel == 3)
-			{
-				zoomLevel = 0;
-			}
+			zoomLevel = Maths::Clamp(zoomLevel - 1, 0, 8);
 
 			Tap(this);
 		}
 		else  if (controls.isKeyJustPressed(controls.getActionKeyKey(AK_ZOOMIN)))
 		{
-			if (zoomLevel == 0)
-			{
-				zoomLevel = 3;
-			}
-			else if (zoomLevel == 3)
-			{
-				zoomLevel = 2;
-			}
-			else if (zoomLevel == 1)
-			{
-				zoomLevel = 2;
-			}
+			zoomLevel = Maths::Clamp(zoomLevel + 1, 0, 8);
 
```

Comment:
Probably change clamp into min/max, or keep clamp and get rid of the double if statement.

Reply:
Thanks, made the change as requested.